_inc/client/state/site/reducer.js

```diff
@@ -32,6 +35,8 @@ export const data = ( state = {}, action ) => {
 			return merge( {}, state, { site: { features: action.siteFeatures } } );
 		case JETPACK_SITE_PLANS_FETCH_RECEIVE:
 			return merge( {}, state, { sitePlans: action.plans } );
+		case JETPACK_PRODUCTS_FETCH_RECEIVE:
```

Comment:
I'm not sure why this is using the site state tree. It seems like the products should be in their own state tree. Keeping them here kinda makes sense for plans, because plans can vary on a per site basis, but not for products - at least in calypso, those are site-agnostic.

Reply:
I did this to keep them close to plans since they are so closely related. The other option is to create a new reducer and put it elsewhere in the state tree which I am happy to do, but I felt that this was too small to warrant that.

Reply:
I understand how they are related, but keeping them in the site tree doesn't make any sense because not only products are site-agnostic, but they are also not related with sites in any way, and keeping them in the state tree can confuse developers that they are related in some way. Can we change that?

Reply:
Ok let's change that, but in a follow up PR to get testing started soon.

Reply:
Tracked in #14038